bin/cloudgov/get-s3-access

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+usage="
+$0: Gives user access to s3.
```

Comment:
Let's be more specific: 
Generates AWS credentials and uses "aws configure" to store them, to provide the user with s3 access.